src/test/Makefile

```diff
@@ -51,8 +50,8 @@ CXX_FLAGS = $(COMMON_FLAGS) \
 GCOV_CC_FLAGS := -fprofile-arcs -ftest-coverage
 GCOV_L_FLAGS  := -lgcov
```

Comment:
`-fprofile-arcs` implies `-lgcov` when linking, so `lgcov` can be safely removed

And even better would be `--coverage` argument - it is supported both in clang and gcc and enables -fprofile-arcs -ftest-coverage/ -lgcov as necessary ..